motion_computation/src/psm_to_external_object_convertor.cpp

```diff
@@ -204,25 +238,25 @@ std::vector<geometry_msgs::msg::Pose> sample_2d_path_from_radius(const geometry_
 
     tf2::Quaternion quat;
     quat.setRPY(0, 0, turning_angle);
-    sample_pose.orientation.x = quat.x();
-    sample_pose.orientation.y = quat.y();
-    sample_pose.orientation.z = quat.z();
-    sample_pose.orientation.w = quat.w();
+    // sample_pose.orientation.x = quat.x();
+    // sample_pose.orientation.y = quat.y();
+    // sample_pose.orientation.z = quat.z();
+    // sample_pose.orientation.w = quat.w();
```

Comment:
Why is this commented out. Fairly certain that invalidates the math

Reply:
This is commented out because sample pose is being declared below this. And the orientation is assigned a different value there

Reply:
Ah I see now.

Please delete this then